build/v1/types.go

```diff
@@ -86,6 +86,13 @@ type CommonSpec struct {
 	// are ignored.
 	// +optional
 	NodeSelector OptionalNodeSelector `json:"nodeSelector" protobuf:"bytes,9,name=nodeSelector"`
+
+	// mountProxyTrustedCA bind mounts the cluster's trusted certificate authorities,
+	// as defined in the cluster's proxy configuration, into the build. When this field is set to
+	// true, the contents of `/etc/pki/ca-trust` within the build are managed by the build
+	// container and cannot be modified during the build (example - via a Dockerfile `RUN`
+	// instruction).
+	MountProxyTrustedCA *bool `json:"mountProxyTrustedCA,omitempty" protobuf:"varint,10,opt,name=mountProxyTrustedCA"`
```

Comment:
Pointer is used here in the event we want to support build defaults, which requires us to determine user intent.

Reply:
i wonder if we should just call this MountTrustedCA?  we have been a bit wishy-washy on the concept, in some places we call it trusted CA, in others proxy CA.  What are the other APIs a user/admin interacts with that would setup/populate the CAs that ultimately get mounted here?